style.css

```diff
@@ -1,69 +1,138 @@
-h1 {
-  color: #fff;
-  text-align: center;
-  padding-top: 20px;
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@361;400;600&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
 }
-body {
-	height: 100%;
+
+:root {
+  --white-color: #ffffff;
+  --purple-color: #451d63;
+  --green-color: #00cd55;
+  --darkbg: #111c39;
+  --purple-bright: #f501a4;
 }
 
 body {
-	margin: 0;
-	background: linear-gradient(45deg, #49a09d, #5f2c82);
-	font-family: sans-serif;
-	font-weight: 100;
+  background-image: url('./assets/bg.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  font-family: 'Inter';
+  padding-top: 10vh;
 }
 
-.container {
-	position: absolute;
-	top: 50%;
-	left: 50%;
-	transform: translate(-50%, -50%);
+h1 {
+  font-size: clamp(30px, 3.125rem + 8.333333vw, 1.5rem);
+  color: var(--white-color);
+  margin-bottom: 30px;
+  letter-spacing: 2px;
+  font-weight: 700;
+}
+
+.leaderboard {
+  max-width: 970px;
+  margin: 0 auto;
+  text-align: center;
 }
 
 table {
-	width: 1000px;
-	border-collapse: collapse;
-	overflow: hidden;
-	box-shadow: 0 0 20px rgba(0,0,0,0.1);
+  border-collapse: collapse;
+  margin: 0 auto;
+}
+
+img.leaderboard__img {
+  width: 190px;
+  margin-bottom: 45px;
+}
+
+table.dataframe {
+  margin: auto;
 }
 
+table.dataframe,
 th,
 td {
-	padding: 15px;
-	background-color: rgba(255,255,255,0.2);
-	color: #fff;
+  border: none;
 }
 
 th {
-	text-align: left;
+  color: #9076a5;
+  text-align: left;
+  padding: 10px 25px;
+  font-size: 12px;
 }
 
 thead {
-	th {
-		background-color: #55608f;
-	}
+  background-color: var(--purple-color);
+  font-size: 13px;
+  text-transform: uppercase;
+  margin-bottom: 20px;
+}
+
+td {
+  color: var(--white-color);
+}
+
+tbody tr:hover {
+  cursor: pointer;
+  background-color: #ffffff1f !important;
```

Comment:
Can it be done without using `!important`?

Reply:
Ok I will check on this one, just wanted to override previous ones.